src/tulip/infos.ts

```diff
@@ -708,13 +740,42 @@ export class ReserveInfoWrapper implements IReserveInfoWrapper {
 export class VaultInfoWrapper implements IVaultInfoWrapper {
   constructor(public vaultInfo: types.VaultInfo) {}
 
-  getAPY() {
+  getAPR() {
     const numberOfPeriods = 8760; // yearly compound times = 24 * 365
     const apr = (Math.pow(this.vaultInfo.apy / 100 + 1, 1 / numberOfPeriods) - 1) * 100 * 24 * 365;
 
     return Number(apr.toFixed(2));
   }
 
+  getAPY(): number {
```

Comment:
What's the difference btw `getAPY` and `getAPR` ?

Reply:
Well... `APR` and `APY` are different due to the compound period. I think we only need `APY` if we want the value same as Tulip's website.

BTW the `APY` we get right now is only `Fees` part. They have another part called `Yield`, we can't get from their API.

Reply:
1. `getAPR` seems to compound the value from `this.vaultInfo.apy`. Just wonder is the result closed to what Tulip has on their UI? (including the yield part)
2. Noted

Reply:
No, I think it's not possible since the `Fees` is generated from trading volume and `Yield` is from farming.